packages/terra-button-group/src/terra-dev-site/doc/button-group/ButtonGroup.doc.jsx

```diff
@@ -45,6 +47,11 @@ const DocPage = () => (
         example: <ButtonGroupDisabledButtons />,
         source: ButtonGroupDisabledButtonsSrc,
       },
+      {
+        title: 'Button Block Layout',
```

Comment:
Lets update the title to 'ButtonGroup Is Block - width stretches to fill container'

Reply:
Updated!